### DISCUSSION TITLE
Add repo issues link to package descriptions

### DISCUSSION THREAD

**Discussion by pauby** [chocolatey-community/chocolatey-packages#1749] [Ideas]
People are posting Disqus questions on community packages that by and large don't get an answer. I feel we should add a note to the bottom of the description of each package to link to this repo's issues page.

I also feel we should add a note to the description stating that these packages are automatically updated and if they are out of date by more than x days, they should raise an issue.

I appreciate there are a huge number of packages in here, some of which have no had updates in some time (as the installer hasn't been updated) but I feel we could make a real dent in the packages that _are_ updated regularly.

Thoughts?

**Comment by AdmiringWorm** [comment]
> People are posting Disqus questions on community packages that by and large don't get an answer. I feel we should add a note to the bottom of the description of each package to link to this repo's issues page.

I think it makes sense to have something in the description, or a pinned comment on the package details page that mentions using this repository for any bugs at least (IMO, a pinned Disqus comment could maybe make more sense than having it in the description).

> I also feel we should add a note to the description stating that these packages are automatically updated and if they are out of date by more than x days, they should raise an issue.

This came up as a suggestion in the past as well, I believe it was decided then that the packages in this repository should be treated as they are normal package instead of automated packages (back then automated packages was mostly considered to be Ketarin packages).

I don't have a preference either way though.

But there is one thing I want to note though. we do have a limit of using maximum 4 000 characters in the description, including either a notice about it being an automated package or a notice to open issues in this repository may push the number of characters over the limit.

**Comment by pauby** [comment]
As discussions have been enabled, I'm going to convert this over to a discussion.
